src/main/twirl/uk/gov/hmrc/play/views/helpers/DateFieldsFreeInlineLegend.scala.html

```diff
@@ -38,7 +38,10 @@
             @args.toMap.get('_inputHint).map { hint =>
                 <span class="form-hint">@hint</span>
             }
-            @formItem.errors.filter(error => fieldNames.contains(error.key)).map { error => <span class="error-notification">@Messages(error.message)</span>}
+            @formItem.errors.filter(error => fieldNames.contains(error.key)).map { error => <span class="error-notification">
+                @if(args.toMap.get('_errorPrefix).isDefined ){<span class="visually-hidden">@args.toMap.get('_errorPrefix) </span>}
```

Comment:
Could the pattern on Line No 38 be used here too (Use map instead of if)?

Reply:
This has been updated as per above.